libs/sdk-ui-kit/src/Header/Header.tsx

```diff
@@ -51,6 +51,15 @@ export interface IHeaderMenuItem {
     target?: string;
     onClick?: (obj: any) => void;
 }
+
+/**
+ * @internal
+ */
+export interface IHeaderBadge {
+    key: string;
```

Comment:
Is key necessary?

Reply:
Unfortunately, it seems to be, since the only other prop is a function we don't have a way to derive the key from it. And I am not sure that using index as a key would be better.

Reply:
And what about `badges: IHeaderBadgeProps[]` ? Or just  `badges: React.ReactNode` (you still should be able to do: `badges={[<HeaderBadge key="1" />, <HeaderBadge key="2" />]}`?  And for "vertical badges", we can add something like `<HeaderBadges vertical>{...badges}</Badges>`? My point is if IHeaderBadge is not redundant?

Reply:
Good point, simplified.